chrome/content/zotero/xpcom/data/collection.js

```diff
@@ -23,7 +23,7 @@
     ***** END LICENSE BLOCK *****
 */
 
-Zotero.Collection = function() {
+Zotero.Collection = function(name) {
```

Comment:
I'm OK with a `params` argument, as an alternative to setting properties after the constructor.